File: app/src/main/java/com/camtech/android/tweetbot/adapters/FragmentAdapter.java
package com.camtech.android.tweetbot.adapters;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

import com.camtech.android.tweetbot.fragments.OccurrencesFragment;
import com.camtech.android.tweetbot.fragments.TweetPostedFragment;

public class FragmentAdapter extends FragmentPagerAdapter {

    public FragmentAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int position) {
        switch (position) {
            case 0:
                return new OccurrencesFragment();
            case 1:
                return new TweetPostedFragment();
            default:
                return null;
        }
    }

    @Override
    public int getCount() {
        return 2;
    }

}

File: app/src/main/java/com/camtech/android/tweetbot/adapters/FragmentAdapter.kt
package com.camtech.android.tweetbot.adapters

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentPagerAdapter

import com.camtech.android.tweetbot.fragments.OccurrencesFragment
import com.camtech.android.tweetbot.fragments.TweetPostedFragment

class FragmentAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {

    override fun getItem(position: Int): Fragment? {
        return when (position) {
            0 -> OccurrencesFragment()
            1 -> TweetPostedFragment()
            else -> null
        }
    }

    override fun getCount() = 2
}